cities.html

```diff
@@ -254,15 +254,15 @@
 								></field>
 								<br />
 								<field title="Population" v-bind:value="city.population">
-									{{ city.population }} people
+									{{ (city.population).toLocaleString('en-US', {valute: 'USD'}) }} people
```

Comment:
I do not think `{valute: 'USD'}` is a valid option and is likely ignored by `toLocaleString`. Are you trying to display currency? If you are simply looking to add commas (or periods if the user is in Europe etc), simply `toLocaleString()` with no parameters suffices.